test/modules/diten/moduleShadowFormal.good

```diff
@@ -1,2 +1,4 @@
-moduleShadowFormal.chpl:9: warning: Module level symbol is hiding function argument 'a'
+moduleShadowFormal.chpl:9: In function 'foo':
+moduleShadowFormal.chpl:10: warning: module-level symbol is hiding function argument 'a'
```

Comment:
These are pre-existing conditions (so don't need to be dealt with here, and are definitely not show-stoppers for making these changes), but while I'm reviewing them:

* "module-level symbol" seems like a weird noun to me here (though I appreciate the introduction of the hyphen! :) ).  It feels more like "symbol made available by module 'use'..." or "symbol made available by 'use M1'..." would be more accurate.
* "function argument": I'd simplify to just "argument" (in part because I'm not keen on "function" as a term anymore, and in part because I don't think it adds additional value).

If I were fine-tuning this error, I'd also wonder whether we could reorganize the lines to present what seems like a clearer picture to me, like:
```bash
moduleShadowFormal.chpl:9: In function 'foo':
moduleShadowFormal.chpl:9: warning: argument 'a' is shadowed
moduleShadowFormal.chpl:11: note: in expressions like this one
moduleShadowFormal.chpl:10: note: by a symbol 'use M1' makes available
moduleShadowFormal.chpl:2: note: that is defined here
```

(where I'm not saying this is perfect or beautiful, but I like that it points to where argument 'a' is defined and where it's used).  This again, is not at all necessary or a showstopper, but since I'm reviewing the messages, figured I'd mention it.